src/main/java/uk/gov/companieshouse/insolvency/delta/mapper/PractitionerAddressMapper.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.companieshouse.insolvency.delta.mapper;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import uk.gov.companieshouse.api.delta.PractitionerAddress;
+
+@Mapper(componentModel = "spring")
+public interface PractitionerAddressMapper {
+
+    @Mapping(target = "addressLine2",
+            expression = "java(emptyStringToNull(sourcePractitionerAddress.getAddressLine2()))")
+    @Mapping(target = "locality",
+            expression = "java(emptyStringToNull(sourcePractitionerAddress.getLocality()))")
+    @Mapping(target = "region",
```

Comment:
I am not sure we need this? ideally we should control by JsonInclude NON_EMPTY on objects.

Reply:
Tried implementing this, but it looks like it does not work in combination with mapstruct - it does not get picked up when autogenerating the mapstruct implementations. The unit tests also fail: 
```
[ERROR]   InsolvencyMapperTest.shouldMapInsolvencyToCompanyInsolvency:81 
expected: null
 but was: ""
```

We have to use the expression syntax for now.

Reply:
Just to add this was also motivated by the example target data which does not include empty strings in the address.